sphinx/ext/autosummary/generate.py

```diff
@@ -320,11 +320,34 @@ def generate_autosummary_content(name: str, obj: Any, parent: Any,
         ns['members'] = dir(obj)
         ns['inherited_members'] = \
             set(dir(obj)) - set(obj.__dict__.keys())
+
         ns['methods'], ns['all_methods'] = \
             _get_members(doc, app, obj, {'method'}, include_public={'__init__'})
         ns['attributes'], ns['all_attributes'] = \
             _get_members(doc, app, obj, {'attribute', 'property'})
 
+        method_string = [m.split(".")[-1] for m in ns['methods']]
+        attr_string = [m.split(".")[-1] for m in ns['attributes']]
+
+        # Find the first link for this attribute in higher classes
+        ns['inherited_paths'] = []
+        ns['inherited_methods'] = []
+        ns['inherited_attributes'] = []
+
+        for i in (set(dir(obj)) - set(obj.__dict__.keys())):
```

Comment:
Not all classes have a `__dict__` (and sometimes there are weird ones). So I think you should look at the implementation of `_get_members` to see how members are looked-up.